README.md

```diff
@@ -1,20 +1,11 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Microverse - Portfolio
-
-> this branch is for the implementation of
->
-> **Works section**
->
-> **About myself section**
->
+This is the code implementation for portfolio template #1
 
 
 ## How to?
 
-**Important**: please remember to inspect the design in a mobile
-version size. i.e. `tested in 400px width`
-
 > `clone this repo`
 >
 > open `index.html`
```

Comment:
- [ ] Great on making some changes on the README file. Its main requirement is to phase a standard readme file. Kindly, make sure all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a student's repo.  That includes :-
- Project Name
- Built With
- Live Demo (if available)
- Getting Started
- Authors
-  Contributing
- Show your support
- Acknowledgments
- License
- > Note: you may leave some details empty if yet to be implemented. Like the `live demo` and `licence`

Reply:
Done! Thank you @Tadesse-Alemayehu